source/_includes/header.html

```diff
@@ -65,6 +65,7 @@
                 <li><a href="/tag-developers/">Tag Developers Guide</a></li>
                 <li><a href="/maven-archetypes/">Maven Archetypes</a></li>
                 <li><a href="/plugins/">Plugins</a></li>
+                <li><a href="/maven/">Maven-generated site</a></li>
```

Comment:
This duplicates already existing entry, see `Project Info` under `Support`

Reply:
Yes, it does.

Should that item under Support be called "Maven Project Info" ?....

Reply:
👍